cmds_init

```diff
@@ -0,0 +1,29 @@
+docker run -d --name animals_db -p :32771 -e POSTGRES_USER=lina -e POSTGRES_PASSWORD=cat -e POSTGRES_DB=animals postgres
```

Comment:
what is this file for? 
there are:
docker run command, 
sql commands for db,
some constants for ... what?
i don't get it.